README.md

```diff
@@ -242,6 +242,29 @@ while (preparedStatement.fetch())
 }
 ```
 
+### Convenient read functions
+**psReadQuery**
+```c++
+psReadQuery(preparedStatement, <Callable>)
+```
+or
+```c++
+psReadQuery("<SELECT query string>", connection, <Callable>)
+```
+Where callable can be C function, lambda, or member function, however in the last case you need to use
```

Comment:
Please add some examples. 
Also describe what happens when wrong method, or wrong variables for result binding is provided.